aplib.net-demo/Assets/Scripts/Inventory/AmmoPouch.cs

```diff
@@ -0,0 +1,46 @@
+using UnityEngine;
+using UnityEngine.Serialization;
+
+/// <summary>
+/// Class representing an Ammo Pouch.
+/// </summary>
+public class AmmoPouch : MonoBehaviour
+{
+    [FormerlySerializedAs("MaxAmmoCount")] [SerializeField]
+    public int maxAmmoCount;
+
+    private int _currentAmmoCount;
+
+    private void Start() => _currentAmmoCount = maxAmmoCount;
+
+    /// <summary>
+    /// Adds ammunition to the ammo pouch.
+    /// </summary>
+    /// <param name="ammo">The amount of ammunition to add.</param>
+    public void AddAmmo(int ammo)
+    {
+        Debug.Log("Add ammunition to pouch!");
```

Comment:
It would be really nice if you could display the current ammo count in the HUD instead of using debug logs, but maybe out of scope for this PR.

Reply:
That is indeed out of scope and should is mentioned in [AP-150](https://zomsa.atlassian.net/browse/AP-150)